SpouseAndParentSharesCalculation.py

```diff
@@ -0,0 +1,87 @@
+import AreFatherMotherWifeAlive as cp
+import ApplicationOutput as op
```

Comment:
Abbreviations...!! I am not going to mention it again

Reply:
Ok..